Author: Clemens Ladisch <clemens@ladisch.de>
Description: libdvbv5: T2 delivery descriptor: fix wrong size of bandwidth field
Origin: https://patchwork.linuxtv.org/patch/40523/
Bug-Debian: http://bugs.debian.org/859008

--- a/lib/include/libdvbv5/desc_t2_delivery.h
+++ b/lib/include/libdvbv5/desc_t2_delivery.h
@@ -86,7 +86,7 @@
 			uint16_t transmission_mode:3;
 			uint16_t guard_interval:3;
 			uint16_t reserved:2;
-			uint16_t bandwidth:3;
+			uint16_t bandwidth:4;
 			uint16_t SISO_MISO:2;
 		} __attribute__((packed));
 	} __attribute__((packed));
